src/number_format.js

```diff
@@ -21,20 +21,23 @@ const propTypes = {
     PropTypes.number,
     PropTypes.string
   ]),
+  allowNegative: PropTypes.bool,
   customInput: PropTypes.func
 };
 
 const defaultProps = {
   displayType: 'input',
   decimalSeparator: '.',
-  decimalPrecision: false
+  decimalPrecision: false,
+  allowNegative: true
```

Comment:
I deal with negative liquidity every day so it seemed like a reasonable default financial use cases, this can be changed to false